src/billable-services/billable-service-form/billable-service-form.workspace.tsx

```diff
@@ -233,8 +234,8 @@ const BillableServiceFormWorkspace: React.FC<BillableServiceFormWorkspaceProps>
     setIsSubmitting(true);
 
     const payload = {
-      name: data.name,
-      shortName: data.shortName || '',
+      name: data.name.trim(),
+      shortName: (data.shortName || '').trim(),
```

Comment:
```suggestion
      name: data.name,
      shortName: data.shortName ?? '',
```

The zod schema's [trim transform](https://zod.dev/api?id=strings) already removes extraneous whitespace from the field input, so trimming it again in the payload is redundant. 

Also, it's more semantically correct to use nullish coalescing (`??`) instead of `||` for the `shortName` fallback. The `||` operator treats all falsy values (empty strings, `0`, `false`) as falsy, while `??` only handles `null` and `undefined`. Since `shortName` is an optional string field, `??` more accurately represents the intent: "use empty string only if the value is nullish, not if it's an intentionally empty string."